src/requirements.txt

```diff
@@ -1,27 +1,11 @@
-# Other Python modules installed locally by the local installation script:
-# pandas
-# sklearn
-# clustergrammer_widget
-beautifulsoup4==4.12.3
-black==24.1.1
-coverage==7.4.1
-cryptography==42.0.2
-decorator==5.1.1
-flake8==7.0.0
-html5lib==1.1
-idna==3.6
-isort==5.13.2
+clustergrammer_widget==1.13.3
+ipython==8.10.0
+ipywidgets==7.7.1
 jinja2==3.1.3
-jupyter-console==6.6.3
-markupsafe==2.1.4
-ndg-httpsclient==0.5.1
-pygments==2.17.1
+notebook==6.5.6
+pandas==2.2.2
 pymongo==4.6.1
-pyopenssl==24.0.0
-pytest==8.0.0
-pytest-cov==4.1.0
+pyyaml==6.0
```

Comment:
This should be 6.0.1 (it used to be in its own file so I remember the version...).